tests/util/test_generate_gwl_tables.py

```diff
@@ -0,0 +1,24 @@
+"""Tests capability to read CMIP6 simulation information from aws"""
+
+import pandas as pd
+import pytest
+
+from climakitae.util.generate_gwl_tables import get_sims_on_aws
+
+
+def test_get_sims_on_aws():
+    """Check that expected scenarios and models are returned."""
+    df = pd.read_csv("https://cmip6-pds.s3.amazonaws.com/pangeo-cmip6.csv")
+    df_subset = df[
```

Comment:
I'm not sure, but I don't think `df_subset` is ever used?

Reply:
Removed